Cargo.toml

```diff
@@ -1,22 +1,26 @@
-[package]
-name = "bad"
-authors = [ "ThePhD", "mcy" ]
-version = "0.0.0"
-edition = "2021"
-keywords = [ "bad", "b", "compiler", "shitpost", "shitcode" ]
-license = "CC0-1.0"
-
-[lib]
-name = "bad"
-path = "bad/source/lib.rs"
-
-
-[[bin]]
-name = "badc"
-path = "badc/source/main.rs"
-
-
-[dependencies]
-bumpalo = "3.11.0"
-nom = "*"
-logos = "*"
+[package]
+name = "bad"
+authors = [ "ThePhD", "mcy" ]
+version = "0.0.0"
+edition = "2021"
+keywords = [ "bad", "b", "compiler", "shitpost", "shitcode" ]
+license = "CC0-1.0"
+
+[lib]
+name = "bad"
+path = "bad/source/lib.rs"
+
+
+[[bin]]
+name = "badc"
+path = "badc/source/main.rs"
+
+
+[dependencies]
+[dependencies.bumpalo]
+version = ">=0,<99999999999999"
```

Comment:
`= "*"` is probably what you want. :P

Reply:
="*" *is* what I want, but if I do that then apparently cargo will be all >:( angry-face about it if we ever upload this. (We probably won't but I'm doing this on pricinple; I can probably change it back. :D)